app/src/main/java/com/snaggly/ksw_toolkit/gui/SystemTwaks.kt

```diff
@@ -48,6 +53,12 @@ class SystemTwaks(val coreServiceClient: CoreServiceClient) : Fragment() {
         initElements()
         initButtonClickEvents()
         autoThemeToggle.requestFocus()
+        if (Build.VERSION.SDK_INT >= 30) {
+            hideTopBarSwitch.isChecked = false
+            hideTopBarTxt.setTextAppearance(R.style.disabledSwitchTextStyle)
+            shrinkTopBarSwitch.isChecked = false
+            shrinkTopBarTxt.setTextAppearance(R.style.disabledSwitchTextStyle)
```

Comment:
Probably also need to set to disabled so it's unclickable. ...Or better just hide it alltogether so, people who bought the later models don't feel they're missing out on something, although that's just a suggestion.

Reply:
After A11, I changed the items to be deleted.
People would care if they couldn't click on them, but they wouldn't care if they weren't displayed from the beginning.
I couldn't figure out how to redraw GridVew and the space was weird, so I moved these items to the bottom
Please point out again if there are any problems.
![Screenshot (2022 12 24 19 37 25)](https://user-images.githubusercontent.com/119941037/209432638-f49265a6-61fb-4d04-958a-5e1912d06f59.png)
![Screenshot (2022 12 24 19 37 25) (2)](https://user-images.githubusercontent.com/119941037/209432642-e2c1607d-6470-431b-a60d-1172a83f3163.png)